### PR TITLE
Track user tenant information

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Add tracking of tenant edition and license. Use special names of the edition, similar to Mist.
Implements http://teampulse.telerik.com/view#item/293312

### CODE REVIEW

File: lib/login.ts [hunk 1]
```diff
@@ -14,10 +14,20 @@ export class UserDataStore implements IUserDataStore {
 	private cookies: IStringDictionary;
 	private user: any;
 
+	private static APPBUILDER_EDITION_TYPE: IStringDictionary = {
```
Comment (ligaz) [inline lib/login.ts]: Can those taken from the server?

Comment (rosen-vladimirov) [inline lib/login.ts]: Unfortunately no :(
We receive only the enum from the Server (https://github.com/Icenium/Ice/blob/master/Common/Common.Data/Authentication/EditionType.cs) while we have to send the "Description" of each item. Same logic is implemented in Mist:
https://github.com/Icenium/Ice/pull/3854/files

Comment (ligaz) [inline lib/login.ts]: We can definitely change this on the server then.

**Comment by teobugslayer** [comment]
:+1: But we may consider adding a new APi on the server to give all clients consistent names for these./

**Pull request merged**